jre_emul/android/platform/libcore/ojluni/src/main/java/java/util/zip/ZipOutputStream.java

```diff
@@ -44,9 +45,24 @@
 public
 class ZipOutputStream extends DeflaterOutputStream implements ZipConstants {
 
+    /**
+     * Whether to use ZIP64 for zip files with more than 64k entries.
+     * Until ZIP64 support in zip implementations is ubiquitous, this
+     * system property allows the creation of zip files which can be
+     * read by legacy zip implementations which tolerate "incorrect"
+     * total entry count fields, such as the ones in jdk6, and even
+     * some in jdk7.
+     */
+    // Android-changed: Always allow use of Zip64.
+    private static final boolean inhibitZip64 = false;
+    //  Boolean.parseBoolean(
```

Comment:
Is this J2ObjC-specific for already there in libcore?

Reply:
this is already there in libcore, and no changes are made specifically for J2ObjC